src/shared/utils/helpers/format-past-date.ts

```diff
@@ -2,13 +2,13 @@ import formatDistanceStrict from "date-fns/formatDistanceStrict";
 import parseISO from "date-fns/parseISO";
 
 export default function (dateString?: string) {
-  if (!dateString) {
-    console.error(err);
-    return "DATE ERROR";
+  if (!dateString || dateString === undefined) {
```

Comment:
`undefined` is already falsey. The or is not needed.

Reply:
It is needed I believe due to the function parameter _for some reason_ being optional? 

`dateString?:`

I'll remove the `?` from the parameter and the right side `or` check.

Reply:
The arg being optional in not a problem. Refer to this [small demo](https://www.typescriptlang.org/play?#code/GYVwdgxgLglg9mABMOcAUBDATgcwPwBciAzlFjGDgJSIDeAUIk4jMImgITbV2PP8QExOABsApgDoRcHGgDkAOTiJucqnyYBfRGJHExvfgKGjJ02dyoBuDYk3179FOmtA). You will see that when an optional arg is not passed (and is thus equal to `undefined`), `!arg` evaluates to `true`. This is because `undefined` is falsey.